README.md

```diff
@@ -6,8 +6,8 @@
 
 ## Prerequisites
 
-1. `Node.js` version >= 14
-2. `NPM` version >= 7
+1. `Node.js` version >= 18
```

Comment:
Let's use just link, see for example https://github.com/project-ncl/pnc-web-ui-react/blob/main/README.md#prerequisites